.github/workflows/ci.yml

```
@@ -26,10 +25,14 @@ jobs:
   test:
     name: Run Tests
     runs-on: ${{ matrix.os }}
+    # Python 3.15 only on main branch push (not PRs) to save time
+    if: |
```

actionlint (1.7.10)

context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)